README.md

```diff
@@ -79,6 +79,11 @@ PLUGINS_CONFIG = {
 }
 ```
 
+To add the required `netbox-acls` tables to your NetBox database, run the `migrate` manager subcommand in your NetBox directory:
```

Comment:
Should the user be informed that they also need to activate the virtual environment in order to successfully run this command?

Reply:
Yes it should, thanks.